File: src/main/java/com/dulkirmod/commands/EnchantRuneCommand.java
package com.dulkirmod.commands;

import com.dulkirmod.Settings;
import net.minecraft.client.Minecraft;
import net.minecraft.command.CommandException;
import net.minecraft.command.ICommandSender;
import net.minecraft.util.ChatComponentText;
import net.minecraft.util.EnumChatFormatting;

public class EnchantRuneCommand extends ClientCommandBase {

    public EnchantRuneCommand() {
        super("enchantrune");
    }

    @Override
    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
        Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(
                EnumChatFormatting.BLACK+ "" + EnumChatFormatting.BOLD + "TOGGLED."));
        Settings.EnchantRune = !Settings.EnchantRune;
    }
}

File: src/main/kotlin/dulkirmod/command/EnchantRuneCommand.kt
package dulkirmod.command

import dulkirmod.DulkirMod.Companion.config
import dulkirmod.utils.TextUtils
import net.minecraft.command.CommandException
import net.minecraft.command.ICommandSender

class EnchantRuneCommand : ClientCommandBase("enchantrune") {
    @Throws(CommandException::class)
    override fun processCommand(sender: ICommandSender, args: Array<String>) {
        config.hideEnchantRune = !config.hideEnchantRune
        TextUtils.toggledMessage("Enchant Rune Hider", config.hideEnchantRune)
        config.save()
    }
}